lib/middleware/check-progress.js

```diff
@@ -18,7 +18,7 @@ module.exports = function checkProgress(route, controller, steps, start) {
         var potentialPaths = _.pluck(controller.options.forks, 'target')
             .concat(controller.options.next);
 
-        var nextStep = controller.getNextStep(req, res).replace(/\/edit$/, '');
+        var nextStep = controller._getForkTarget(req, res);
```

Comment:
Another thought, which I think may be nicer, would be to use the hmpo-form-controller form prototype method directly, rather than the derived class, this would mean the correct method would always be called regardless if it is overridden in a child controller anywhere.

```
FormController.prototype._getForkTarget.apply(controller, arguments);
```